tests/conftest.py

```diff
@@ -555,6 +556,11 @@ def tf_full_version(tensorflow_training_latest_version, tensorflow_inference_lat
     Fixture exists as such, since TF training and TFS have different latest versions.
     Otherwise, this would simply be a single latest version.
     """
+    if Version(tensorflow_training_latest_version) in SpecifierSet(">=2.16"):
```

Comment:
Nit: you can prob just assign a variable for the version object so you don't have to reinstantiate multiple times

Reply:
Thanks, made the change